computer_oriented_statistical_methods/false-position.cpp

```diff
@@ -0,0 +1,35 @@
+#include<stdlib.h>
+#include <math.h>
+#include <iostream>
+float eq(float i) {
+    return (pow(i, 3) - (4 * i) - 9);  // origial equation
+}
+int main() {
+    float a, b, z, c, m, n;
+    system("cls");
```

Comment:
__system("cls");__ will raise __-bash: cls: command not found__ on Linux and macOS.  This must be __system("clear");__ on Linux and macOS.

Reply:
Does __clear__ work on Windows?